mxcube/.gitignore

```diff
@@ -0,0 +1,7 @@
+/activedirectory.yml
+/ca.crt
+/cfg-maxiv-mxcubecore/
+/mxcube3/
+/mxcubecore/
```

Comment:
This is specific for your set-up. I don't have these files and directories!

Reply:
Ah right, there was a mix-up on my side. Fixed.